pagarme/former-kit-skin-pagarme
# Update font family

## Context

This PR aims to change the used `font-family` to `Rubik`family because of the current `font-family` problems.
For this PR you should validate the `font-family` used, colors and `font-weight` from components like: `button`, `form fields`, `input` and `table`. 

## Linked Issues
- [x] Resolves #185 

## Related to
https://github.com/pagarme/former-kit/pull/294

## How to test
To test this PR you should follow the following steps:

1 Clone this repository:
Using a terminal run the following code.
```
git clone git@github.com:pagarme/former-kit-skin-pagarme.git
```

2 Navigate to the created folder
```
cd former-kit-skin-pagarme
```

3 install the dependencies(we use [yarn](https://yarnpkg.com/en/) as the package manager).
```
yarn
```

3 Use [yarn](https://yarnpkg.com/en/) to create a link to this project.
```
yarn link
```
This command should return a message like the following one:
![image](https://user-images.githubusercontent.com/1773766/62299205-e8edf800-b44a-11e9-936d-7007fe845cad.png)

Now with the link to `former-kit-skin-pagarme` created we need to link it.

4 Get back one folder back
```
cd ..
```
5 Clone the `former-kit` project.
```
git clone git@github.com:pagarme/former-kit.git
```

6 Go to `former-kit` installation folder.
```
cd former-kit
```

7 Link the `former-kit` with `former-kit-skin-pagarme`.
```
yarn link former-kit-skin-pagarme
```

8 Install `former-kit` dependencies
```
yarn
```

9 Run `former-kit`'s Storybook
```
yarn storybook
```
This command should trigger a new browser window with all the `former-kit` examples using the `formerr-kit-skin-pagarme` style.
![image](https://user-images.githubusercontent.com/1773766/62299844-1dae7f00-b44c-11e9-8763-552b79a1951a.png)
